exercises/concept/chrono-realms/.meta/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "authors": [
+    "<your_gh_username>",
```

Comment:
```suggestion
```

Exercise is already authored. Any subsequent updates by someone not an author would make them a contributor

Reply:
I think anyone implementing the rest of this exercise could be considered a co-author, see https://github.com/exercism/cairo/pull/272#discussion_r1777017124